### PR TITLE
Remove Push module from AppCenter

### PR DESCRIPTION
**Pull request by vvechkanov**
## Description

- Remove modules

## Related PRs or issues

[AB#83308](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/83308)

### CODE REVIEW

**Review by MatkovIvan — CHANGES_REQUESTED** [review]
Update changelog

**Comment by vvechkanov** [comment]
Rebased on new develop.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # App Center SDK for Unity Editor Extensions Change Log
 
+## Version 3.3.2 (Under development)
```
Comment (MatkovIvan) [inline CHANGELOG.md]: Why 3.3.2? It's like 2.1.0 or so

**Review by AnastasiaKubova — DISMISSED** [review]

**Review by olmisa — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
 # App Center SDK for Unity Editor Extensions Change Log
 
-## Release 2.0.1 (in development)
+## Version 2.1.0 (Under development)
+
+### App Center Push
```
Comment (olmisa) [inline CHANGELOG.md:8]: Maybe this change should be marked as 'Breaking change' as it was done for 'Auth' module in previous major version?

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**